cmd/minishift/cmd/start.go

```diff
@@ -319,29 +243,15 @@ func automountHostfolders(driver drivers.Driver) {
 func calculateDiskSizeInMB(humanReadableDiskSize string) int {
 	diskSize, err := units.FromHumanSize(humanReadableDiskSize)
 	if err != nil {
-		atexit.ExitWithMessage(1, fmt.Sprintf("Disk size is not valid: %s", err))
+		atexit.ExitWithMessage(1, fmt.Sprintf("Disk size is not valid: %v", err))
 	}
 	return int(diskSize / units.MB)
 }
 
-// init configures the command line options of this command
-func init() {
-	initStartFlags()
-	initClusterUpFlags()
-	initSubscriptionManagerFlags()
-
-	startCmd.Flags().AddFlagSet(startFlagSet)
-	startCmd.Flags().AddFlagSet(clusterUpFlagSet)
-	startCmd.Flags().AddFlagSet(subscriptionManagerFlagSet)
-
-	viper.BindPFlags(startCmd.Flags())
-	RootCmd.AddCommand(startCmd)
-
-	provision.SetDetector(&provisioner.MinishiftProvisionerDetector{Delegate: provision.StandardDetector{}})
-}
+// initStartFlags creates the CLI flags which needs to be passed on to 'lib-machine'
```

Comment:
`lib-machine` to `libmachine`. I think `libmachine` is more familiar than `lib-machine`.